go-controller/pkg/ovnwebhook/podadmission.go

```diff
@@ -123,9 +134,7 @@ func (p PodAdmission) ValidateUpdate(ctx context.Context, oldObj, newObj runtime
 		}
 	}
 
-	// All the checks beyond this point are ovnkube-node specific
-	// If the user is not ovnkube-node exit here
-	if !isOVNKubeNode {
+	if !isOVNKubeNode && csrAcceptanceCondition == nil {
```

Comment:
This condition means that if there is an additional acceptance condition we will do all of the checks below.
Could you add back the comment stating that?
Additionally the error messages mentioning `ovnkube-node` as the user are no longer fully accurate.

Reply:
Changed.